### ISSUE TITLE
Multiple URLs in Trigger Actions fail

### ISSUE THREAD

**Issue by xanthos84** [VorlonCD/bi-aidetection#230]
hi, I am using the AiTool for a long time now,

but after update with version AITool-VorlonCD-v2.0.1014 

the Trigger URLs in Actions do not work correct anymore,

if you enter multiple URLs - sometimes only one, sometimes none trigger.

Can you please check this?

**Comment by Tinbum1** [comment]
I'd update to latest version and try that.

**Comment by VorlonCD** [comment]
I dont think its AITOOL, I had some delayed triggers happening with the last few BI builds.   Perhaps try disabling firewall if the latest version doesnt work?

**Comment by xanthos84** [comment]
Firewall is not active, so that does not change anything.

I changed now the trigger from AITool into BI with the actions as a webrequest,
so I have only one trigger left in AITool and thats working for now.

**Comment by VorlonCD** [comment]
Can you elaborate?   Do you mean you changed code from HttpClient to WebRequest?

And as far as I can tell you only need one BI URL anyway that should do everything?
`[BlueIrisURL]/admin?trigger&camera=[camera]&user=[Username]&pw=[Password]&flagalert=1&memo=[summary]&jpeg=[ImagePathEscaped]`

**Comment by xanthos84** [comment]
I do not changed any code - sorry for the misunderstanding.

You are right about the BI URL and how to trigger BI - thats nothing I have changed,
that behaviour is working correct.
But I am using a Homematic Smart Home system with system variables which are getting triggered/changed with Web Requests by calling an Web API,
and originally I included that call within AITool - but now I have moved that call to BI actions.

So nothing AITool related.
